src/lumigo_opentelemetry/instrumentations/psycopg/instrumentation.py

```diff
@@ -0,0 +1,258 @@
+# mypy: ignore-errors
```

Comment:
copied in from opentelemetry-python-contrib, the only fix necessary was pg_cursor (psycopg.extensions.cursor) -> psycopg.Cursor

Reply:
double checking the license - the previous comment needs to be in the file itself